cli/index.ts

```diff
@@ -653,19 +642,13 @@ export function runCli() {
         }
       },
       {
-        format: `listtables <${ProjectConfigOptions.warehouse.name}>`,
-        description: "List tables on the configured data warehouse.",
-        positionalOptions: [ProjectConfigOptions.warehouse],
+        format: `listtables`,
```

Comment:
up to you but: i think we should get rid of these CLI commands, i bet they are effectively never used